pubpub/pubpub
# update: Further tweaks to the pub footer (4/x)

We're closing in on replacing all the stuff at the bottom of a pub with the new `PubBottom` component. This PR has some misc. improvements:

- Opens the Comments section by default
- Makes every footer row a click target to expand that row
- Darkens the text color to AA accessibility standards
- Adds the License section to the pub footer
- Correctly wires icons into the `communityData.accentColorDark` color

_Test plan:_
Open [a pub with some stuff in it](http://localhost:9876/pub/xiyvipgb) and...
- Check that the Comments section is open by default
- Check that you can open rows by clicking anywhere on the row
- Check that if you change the community accent colors, the icon colors in the footer change too.

_Screenshot:_
<img width="700" alt="image" src="https://user-images.githubusercontent.com/2208769/63623875-ccb42580-c5c8-11e9-82f1-2cc0151c46f7.png">
